### ISSUE TITLE
[bug] `lodash/chaining` and `toString` false positive and fatal error

### ISSUE THREAD

**Issue by SpainTrain** [wix/eslint-plugin-lodash#115]
In some situations the `lodash/chaining` rule incorrectly flags use of `toString` (lodash's `toString`) as a failure.  E.g., this is taken from a larger file exhibiting the behavior:

```js
import {toString} from 'lodash'
import {map as fpMap} from 'lodash/fp'

const toStringValues = fpMap((option) => ({
    ...option,
    value: toString(option.value),
}))
```

When I tried to create a minimal reproduction, I received a fatal error:

```
import {toString} from 'lodash'

toString(1)
```

```
$ ./node_modules/.bin/eslint app/test.js 
Cannot read property 'parent' of null
TypeError: Cannot read property 'parent' of null
    at getEndOfChain (/home/spainhower/repos/hz-webapp/node_modules/eslint-plugin-lodash/lib/rules/unwrap.js:33:50)
    at EventEmitter.visitors.CallExpression (/home/spainhower/repos/hz-webapp/node_modules/eslint-plugin-lodash/lib/rules/unwrap.js:42:27)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
    at NodeEventGenerator.enterNode (/home/spainhower/repos/hz-webapp/node_modules/eslint/lib/util/node-event-generator.js:40:22)
    at CodePathAnalyzer.enterNode (/home/spainhower/repos/hz-webapp/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:608:23)
    at CommentEventGenerator.enterNode (/home/spainhower/repos/hz-webapp/node_modules/eslint/lib/util/comment-event-generator.js:97:23)
    at Controller.enter (/home/spainhower/repos/hz-webapp/node_modules/eslint/lib/eslint.js:898:36)
    at Controller.__execute (/home/spainhower/repos/hz-webapp/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/home/spainhower/repos/hz-webapp/node_modules/eslint/node_modules/estraverse/estraverse.js:501:28)
```

While debugging this issue I also saw the error mentioned in #113 as well as:

```
TypeError: info.name.replace is not a function
    at buildInfo (/home/spainhower/repos/hz-webapp/node_modules/eslint-plugin-lodash-fp/rules/core/lodash-util.js:17:28)
    at isIdentifierMethod (/home/spainhower/repos/hz-webapp/node_modules/eslint-plugin-lodash-fp/rules/core/lodash-util.js:72:7)
    at isAnyMethod (/home/spainhower/repos/hz-webapp/node_modules/eslint-plugin-lodash-fp/rules/core/lodash-util.js:84:39)
    at Object.<anonymous> (/home/spainhower/repos/hz-webapp/node_modules/eslint-plugin-lodash-fp/rules/core/lodash-util.js:93:20)
    at r (/home/spainhower/repos/hz-webapp/node_modules/lodash/lodash.min.js:5:194)
    at Object.u [as isAnyMethodOf] (/home/spainhower/repos/hz-webapp/node_modules/lodash/lodash.min.js:47:101)
    at EventEmitter.info.merge.CallExpression (/home/spainhower/repos/hz-webapp/node_modules/eslint-plugin-lodash-fp/rules/no-chain.js:17:57)
    at emitOne (events.js:95:20)
    at EventEmitter.emit (events.js:182:7)
    at NodeEventGenerator.enterNode (/home/spainhower/repos/hz-webapp/node_modules/eslint/lib/util/node-event-generator.js:40:22)
    at CodePathAnalyzer.enterNode (/home/spainhower/repos/hz-webapp/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:608:23)
```

Same env as #113

**Comment by SpencerCarstens** [comment]
To add to this, I am also getting a `preferred-alias` error on _.toString().

**Comment by ganimomer** [comment]
Fixed in `v2.1.5`. Thanks for the catch!

**Comment by SpainTrain** [comment]
Thank you for the quick turnaround on the fix!

This appears to have fixed the issue, but now we are seeing a similar false failure

```
/home/ubuntu/hz-webapp/app/components/buyer/offer/compose-helpers.js
   79:12  error  Method 'toString' is an alias, for consistency prefer using 'function toString() { [native code] }'  lodash/preferred-alias
  106:20  error  Method 'toString' is an alias, for consistency prefer using 'function toString() { [native code] }'  lodash/preferred-alias
  109:17  error  Method 'toString' is an alias, for consistency prefer using 'function toString() { [native code] }'  lodash/preferred-alias

/home/ubuntu/hz-webapp/app/components/common/readonly-field.js
  64:71  error  Method 'toString' is an alias, for consistency prefer using 'function toString() { [native code] }'  lodash/preferred-alias
```

I will open a separate issue to track.
